.nyc_output/processinfo/4c253315-330d-4cee-99b8-b914643d1eaa.json

```diff
@@ -0,0 +1 @@
+{"uuid":"4c253315-330d-4cee-99b8-b914643d1eaa","parent":"ccc3554d-98f2-4612-9c6f-2e548914bd12","pid":71778,"argv":["/Users/samuelfialho/.nvm/versions/node/v10.23.0/bin/node","/Users/samuelfialho/nuxeo/doc/documentation/verify/index.js"],"execArgv":["-r","/Users/samuelfialho/nuxeo/doc/documentation/node_modules/esm/esm.js"],"cwd":"/Users/samuelfialho/nuxeo/doc/documentation","time":1635508663175,"ppid":71758,"root":"e83ae20c-623d-409a-ab98-dff5289e39da","coverageFilename":"/Users/samuelfialho/nuxeo/doc/documentation/.nyc_output/4c253315-330d-4cee-99b8-b914643d1eaa.json","externalId":"verify/index.js","files":[]}
```

Comment:
Are all these `.nyc_output/**` files to be commited?

Reply:
🤦  no, no, thanks for catching those

Reply:
👍  No problem, it just felt weird because they point to stuff on your laptop ;)